commands/cloud.go

```diff
@@ -131,5 +149,99 @@ func runCloudSeal(cmd *cobra.Command, args []string) error {
 	fmt.Printf("%s written.\n", outputFile)
 
 	return nil
+}
+
+func downloadKubeSeal() error {
+	releases := "https://github.com/bitnami-labs/sealed-secrets/releases/latest"
+
+	releaseVersion := downloadVersion
+	if len(downloadVersion) == 0 {
+		version, err := findRelease(releases)
+		if err != nil {
+			return err
+		}
+		releaseVersion = version
+	}
+
+	osVal := runtime.GOOS
+	arch := runtime.GOARCH
+
+	if arch == "x86_64" {
+		arch = "amd64"
+	}
+
+	downloadURL := "https://github.com/bitnami/sealed-secrets/releases/download/" + releaseVersion + "/kubeseal-" + osVal + "-" + arch
+
+	fmt.Printf("Starting download of kubeseal %s, this could take a few moments.\n", releaseVersion)
+	output, err := downloadBinary(http.DefaultClient, downloadURL, "kubeseal")
+
+	if err != nil {
+		return err
+	}
 
+	fmt.Printf(`Download completed, please run:
+
+  chmod +x %s
+  %s --version
+  sudo install %s /usr/local/bin/
+
+  `, output, output, output)
+
+	return nil
+}
+
+func findRelease(url string) (string, error) {
```

Comment:
This approach can be used to inform users when there is an update for the `faas-cli` itself. It also doesn't count towards the rate limiting.